[2024-05-16T11:19:50.043+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:19:50.076+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-16T11:19:50.087+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-16T11:19:50.089+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:19:50.099+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-15 00:00:00+00:00
[2024-05-16T11:19:50.105+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-15T00:00:00+00:00', '--job-id', '413', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpwbju5m7k']
[2024-05-16T11:19:50.108+0000] {standard_task_runner.py:91} INFO - Job 413: Subtask import_table2
[2024-05-16T11:19:50.110+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=294) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:19:50.116+0000] {standard_task_runner.py:63} INFO - Started process 310 to run task
[2024-05-16T11:19:50.182+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:19:50.246+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-15T00:00:00+00:00'
[2024-05-16T11:19:50.247+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:19:50.253+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:19:50.255+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql;\n                ']
[2024-05-16T11:19:50.264+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:19:50.325+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:19:50.326+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:19:50.345+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240515T000000, start_date=20240516T111950, end_date=20240516T111950
[2024-05-16T11:19:50.382+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:19:50.424+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:19:50.426+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-16T11:22:33.231+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:22:33.257+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-16T11:22:33.262+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-16T11:22:33.264+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:22:33.274+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-15 00:00:00+00:00
[2024-05-16T11:22:33.280+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=574) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:22:33.286+0000] {standard_task_runner.py:63} INFO - Started process 585 to run task
[2024-05-16T11:22:33.281+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-15T00:00:00+00:00', '--job-id', '467', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpov6gkl55']
[2024-05-16T11:22:33.294+0000] {standard_task_runner.py:91} INFO - Job 467: Subtask import_table2
[2024-05-16T11:22:33.344+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:22:33.413+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-15T00:00:00+00:00'
[2024-05-16T11:22:33.415+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:22:33.424+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:22:33.426+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql\n                ']
[2024-05-16T11:22:33.433+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:22:33.533+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:22:33.537+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:22:33.552+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240515T000000, start_date=20240516T112233, end_date=20240516T112233
[2024-05-16T11:22:33.595+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:22:33.614+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:22:33.615+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-05-16T11:24:50.045+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-16T11:24:50.074+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-16T11:24:50.079+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [queued]>
[2024-05-16T11:24:50.080+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-05-16T11:24:50.090+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): import_table2> on 2024-05-15 00:00:00+00:00
[2024-05-16T11:24:50.097+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_tables', 'import_table2', 'scheduled__2024-05-15T00:00:00+00:00', '--job-id', '507', '--raw', '--subdir', 'DAGS_FOLDER/dag_mysql.py', '--cfg-path', '/tmp/tmpm3lbzlk5']
[2024-05-16T11:24:50.103+0000] {standard_task_runner.py:91} INFO - Job 507: Subtask import_table2
[2024-05-16T11:24:50.098+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=783) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-16T11:24:50.106+0000] {standard_task_runner.py:63} INFO - Started process 803 to run task
[2024-05-16T11:24:50.152+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_tables.import_table2 scheduled__2024-05-15T00:00:00+00:00 [running]> on host 7084a6d88d3b
[2024-05-16T11:24:50.217+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='bses' AIRFLOW_CTX_DAG_ID='dag_tables' AIRFLOW_CTX_TASK_ID='import_table2' AIRFLOW_CTX_EXECUTION_DATE='2024-05-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-15T00:00:00+00:00'
[2024-05-16T11:24:50.219+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-16T11:24:50.227+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-16T11:24:50.228+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '\n                mysql -h host.docker.internal -u root --password=12345678 test < /opt/***/dags/product_category.sql\n                ']
[2024-05-16T11:24:50.235+0000] {subprocess.py:86} INFO - Output:
[2024-05-16T11:24:50.279+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-16T11:24:50.279+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-16T11:24:50.291+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=dag_tables, task_id=import_table2, execution_date=20240515T000000, start_date=20240516T112450, end_date=20240516T112450
[2024-05-16T11:24:50.309+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-16T11:24:50.317+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-16T11:24:50.318+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
